File: ios/Runner/Plugins/ostext/OsTextViewPlugin.swift
import Flutter
import UIKit

 /**
  * OSNative の TextViewプラグイン　Factory登録を行う
  */
public class OsTextViewPlugin: NSObject, FlutterPlugin {
    public static func register(with registrar: FlutterPluginRegistrar) {
        registrar.register(OsTextViewFactory(messenger: registrar.messenger() as? (NSObject & FlutterBinaryMessenger)), withId: "local_plugins/nativetextview")
    }
}

File: android/app/src/main/java/com/nineeyes/jp/flutter_ui_examples/platform_view_example/plugins/ostext/OsTextViewPlugin.java
package com.nineeyes.jp.flutter_ui_examples.platform_view_example.plugins.ostext;

import io.flutter.plugin.common.PluginRegistry;

/**
 * OSNative の TextViewプラグイン　Factory登録を行う
 */
public class OsTextViewPlugin {
    public static void registerWith(PluginRegistry.Registrar registrar) {
        registrar
                .platformViewRegistry()
                .registerViewFactory(
                        "local_plugins/nativetextview", new OsTextViewFactory(registrar.messenger()));
    }
}